src/main/client/player/player-sprite.ts

```diff
@@ -219,13 +225,15 @@ export class PlayerSprite extends Phaser.GameObjects.Container {
          }
       }
 
-      this.updateAmmo();
+      this.updateAmmo(delta);
       this.gun.update();
       // this.debugger.update(this);
    }
 
    getCenterPoint(): Phaser.Math.Vector2 {
-      return new Phaser.Math.Vector2({ x: this.x, y: this.y }).add(VectorUtil.getUpwardVector(this).scale(48));
+      return new Phaser.Math.Vector2({ x: this.x, y: this.y }).add(
+         VectorUtil.getUpwardVector(this).scale(ClientConfig.PLAYER_SPRITE_HEIGHT / ClientConfig.MAP_OUTPUT_SCALE),
```

Comment:
this one is correct, it is scaled by a positive value

Reply:
Yes it is 48/scale. I do the same in negative in other-player-sprite so the player will show up in the same, correct position but the server can now use the center position.